### PR TITLE
new recipe: KBBI-Qt

### PR DESCRIPTION
**Pull request by mazbrili**
KBBI-Qt is dictionary application for indonesian language

### CODE REVIEW

File: app-text/kbbi-qt/kbbi_qt-1.1.recipe [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+2017-2020 Ade Malsasa Akbar
+2017-2020 Azis Pratama
+2017-2020 Billy
+2017-2020 Christian Kurniawan
+2017-2020 HanHan
+2017-2020 La Ode Muh. Fadlun Akbar
+2017-2020 Ramdziana Feri Yustianto
+2017-2020 Rania el-Amina
+2017-2020 Sucipto
+"
```
Comment (Begasus) [inline app-text/kbbi-qt/kbbi_qt-1.1.recipe:14]: Move line 4 up to line 3, add tabs on the next lines to indent

File: app-text/kbbi-qt/kbbi_qt-1.1.recipe [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+HOMEPAGE="https://github.com/bgli/kbbi-qt"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/bgli/kbbi-qt/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="e858907628fb37789187b98dede196993bc7aaec88dbae9bb05683d576ef4aff"
+SOURCE_FILENAME="kbbi-qt-$portVersion.tar.gz"
+SOURCE_DIR="kbbi-qt-$portVersion"
+ADDITIONAL_FILES="kbbi_qt.rdef.in"
```
Comment (korli) [inline app-text/kbbi-qt/kbbi_qt-1.1.recipe:22]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines: ADDITIONAL_FILES comes after PATCHES

File: app-text/kbbi-qt/kbbi_qt-1.1.recipe (hunk 1 above)
Comment (mazbrili) [reply app-text/kbbi-qt/kbbi_qt-1.1.recipe]: ok done

File: app-text/kbbi-qt/kbbi_qt-1.1.recipe (hunk 2 above)
Comment (mazbrili) [reply app-text/kbbi-qt/kbbi_qt-1.1.recipe]: ok done

**Comment by Begasus** [comment]
Could you see if you upstream the patch? @mazbrili

**Comment by mazbrili** [comment]
> Could you see if you upstream the patch? @mazbrili

i 'll try to make it upstream and if it done i'll  update the recipe
thanks for support

**Pull request merged**